dts/bindings/pwm/nxp,kinetis-ftm.yaml

```diff
@@ -28,4 +28,9 @@ properties:
       category: required
       description: required interrupts
       generation: define
+
+"#cells":
```

Comment:
#cells will  sometime in the future clash when a device integrates several cell related bindings (e.g #clock-cells -> #cells, #pwm-cells -> #cells). We should really start using some more specific here like #pwm-cells.

Reply:
> #cells will sometime in the future clash when a device integrates several cell related bindings (e.g #clock-cells -> #cells, #pwm-cells -> #cells). We should really start using some more specific here like #pwm-cells.

Let's open that up as an issue.